pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -87,11 +87,7 @@ start_test "Test that quit does not show INFO by defaults with --logtostderr"
 # that the default logging level is WARNING, so that no INFO messages
 # are printed between the marker and the (first line) error message
 # from quit. Quit will error out because the server is already stopped.
-#
-# Since quit is likely to actually emit warnings, but we are just
-# interested in the absence of INFO messages, remove the warnings from
-# the output.
-send "echo marker; $argv quit --logtostderr 2>&1 | grep -v '^W'\r"
+send "echo marker; $argv quit --logtostderr 2>&1 | grep -vE '^\[IWEF\]\[0-9\]+ .\* grpc: '\r"
```

Comment:
The test is supposed to check no info messages are printed because the default logging level should be WARNING. The grep should not remove them.

Reply:
Ah, good point. Fixed.